showcase/src/app/welcome/pages/reactive-form-errors/demo-reactive-form-errors-page.component.html

```diff
@@ -0,0 +1,163 @@
+								<div>
+									'required' error:
+									<pre>{{ usernameField.getError("required") | json }}</pre>
+								</div>
+								<div *ngIf="usernameField.hasErrors">
+									Errors:
+									<pre>{{ usernameField.errors | json }}</pre>
+								</div>
+							</mat-card-content>
+						</app-card>
+
+						<app-card class="form-field-info" color="accent">
+							<mat-card-title>Password</mat-card-title>
+							<mat-card-content>
+								<div>Has errors: {{ passwordField.hasErrors }}</div>
+								<div>Has 'pattern' error: {{ passwordField.hasError("pattern") }}</div>
+								<div>Is touched? {{ passwordField.hasState("touched") }}</div>
+								<div>
+									'pattern' error:
+									<pre>{{ passwordField.getError("pattern") | json }}</pre>
+								</div>
+								<div *ngIf="passwordField.hasErrors">
+									Errors:
+									<pre>{{ passwordField.errors | json }}</pre>
+								</div>
+							</mat-card-content>
+						</app-card>
+
+						<app-card class="form-field-info" color="accent">
+							<mat-card-title>Confirm password</mat-card-title>
+							<mat-card-content>
+								<div>Has errors: {{ confirmPasswordField.hasErrors }}</div>
+								<div>Has 'required' error: {{ confirmPasswordField.hasError("required") }}</div>
+								<div>Is touched? {{ confirmPasswordField.hasState("touched") }}</div>
+								<div>
+									'required' error:
+									<pre>{{ confirmPasswordField.getError("required") | json }}</pre>
+								</div>
+								<div *ngIf="confirmPasswordField.hasErrors">
+									Errors:
+									<pre>{{ confirmPasswordField.errors | json }}</pre>
+								</div>
+							</mat-card-content>
+						</app-card>
+					</ng-container>
+
+					<app-card
+						*ngIf="showValidationSummary"
+						class="form-validation-messages"
+						[color]="
+							!usernameField.hasErrors && !passwordField.hasErrors && !confirmPasswordField.hasErrors ? 'success' : 'warning'
+						"
+					>
+						<span *ngIf="!usernameField.hasErrors && !passwordField.hasErrors && !confirmPasswordField.hasErrors">
+							No validation errors
+						</span>
+
+						<!-- TODO: IMPORTANT: the ngIf cannot be used here because it causes some issues with the ErrorComponent created by the ngxFormErrors -->
+						<ul [hidden]="!usernameField.hasErrors && !passwordField.hasErrors && !confirmPasswordField.hasErrors">
+							<li [hidden]="!usernameField.hasErrors">
+								<ng-template ngxFormErrors="username"></ng-template>
+							</li>
+
+							<li [hidden]="!passwordField.hasErrors">
+								<ng-template
+									ngxFormErrors="matchingPasswords.password"
+									ngxFormErrorsFieldName="SHOWCASE.FORMS.NGX_FORMS.PASSWORD_ALIAS"
+								></ng-template>
+							</li>
+
+							<li [hidden]="!confirmPasswordField.hasErrors">
+								<ng-template ngxFormErrors="matchingPasswords.confirmPassword"></ng-template>
+							</li>
+						</ul>
+					</app-card>
+				</div>
+			</form>
+		</example-viewer>
+	</section>
```

Comment:
⚠️ don't forget to add the "Reference" links to the NGX Form errors repo... like we do with the UI components for example );